core/settings.py

```
@@ -0,0 +1,676 @@
+
+import os
+import sys
+import warnings
+from datetime import datetime
+from pathlib import Path
+
+import highlight_io
+from configurations import Configuration, values
+from highlight_io.integrations.django import DjangoIntegration
+from loguru import logger
+from redis.retry import Retry
```

Ruff (0.12.2)

`redis.retry.Retry` imported but unused

Remove unused import: `redis.retry.Retry`

(F401)